script/test

```diff
@@ -168,6 +168,17 @@ do_clean() {
     rm -rfv "${OT_BUILDDIR}"
 }
 
+do_cert_unit_tests() {
```

Comment:
Just do_unit?

We can add all subcommand for all tests?

Reply:
Yes, I plan to also add the tests in `thread-cert` into cmake tests and then add subcommand here for all tests. I did this first to split the change, trying to make it small. Shall we add unit tests first or you'd prefer to add all the things in this PR?